libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsVcEvidence.java

```diff
@@ -23,13 +25,23 @@ public class AuditExtensionsVcEvidence implements AuditExtensions {
     @JsonInclude(NON_NULL)
     private final Boolean successful;
 
+    @JsonProperty("vot")
+    @JsonInclude(NON_NULL)
+    private final Vot vot;
+
     public AuditExtensionsVcEvidence(
             @JsonProperty(value = "iss", required = false) String iss,
             @JsonProperty(value = "evidence", required = false) String evidence,
-            @JsonProperty(value = "successful", required = false) Boolean successful)
-            throws JsonProcessingException {
-        this.iss = iss;
-        this.evidence = evidence == null ? null : new ObjectMapper().readTree(evidence);
-        this.successful = successful;
+            @JsonProperty(value = "successful", required = false) Boolean successful,
+            @JsonProperty(value = "vot", required = false) Vot vot)
```

Comment:
If you wanted to avoid adding `null` to all the existing cases then you could consider adding an an extra constructor that delegates to this one. Something like:
```
public AuditExtensionsVcEvidence(
  String iss,
  String evidence,
  Boolean successful) {
  this(iss, evidence, successful, null);
}
```

Alternatively, I wonder if it's worth pulling out a helper/factory method for generating one of these from a SignedJWT - we seem to have several parallel implementations. (Perhaps moving `getExtensionsForAudit` into a helper on `VcHelper`?)

Reply:
Yea, didnt think of that. This has been implemented and the existing cases have been amended